src/plugins/dashboard/public/application/dashboard_app_controller.tsx

```diff
@@ -234,6 +242,42 @@ export class DashboardAppController {
       }
     };
 
+    const showFilterBar = () =>
+      $scope.model.filters.length > 0 || !dashboardStateManager.getFullScreenMode();
+
+    const getNavBarProps = () => {
+      const isFullScreenMode = dashboardStateManager.getFullScreenMode();
+      return {
+        appName: 'dashboard',
+        config: $scope.isVisible ? $scope.topNavMenu : undefined,
+        className: isFullScreenMode ? 'kbnTopNavMenu__noPadding' : undefined,
```

Comment:
Usually with BEM naming, the class name should be describing the element and not the style. That way it's more scalable in case more/different styles need to be changed. So since this class is added because of a certain **state**, it would be more like:
```suggestion
        className: isFullScreenMode ? 'kbnTopNavMenu-isFullScreen' : undefined,
```